WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationView.swift

```diff
@@ -60,17 +60,12 @@ struct TapToPayEducationView: View {
             }
         }
         .interactiveDismissDisabled(viewModel.isInteractiveDismissDisabled)
-        .sheet(isPresented: $viewModel.showingSetUpFlow,
-               onDismiss: viewModel.reloadHasPreviousTapToPayUsage,
-               content: {
-            TapToPaySettingsFlowPresentingView(
-                configuration: viewModel.configuration,
-                siteID: viewModel.siteID,
-                onboardingUseCase: viewModel.cardPresentPaymentsOnboardingUseCase)
-        })
```

Comment:
Yes. Instead of duplicating `TapToPaySettingsFlowPresentingView` configuration and presentation, I'm passing a binding to `TapToPayEducationView` which presents this same view on the Payments view.